cmd/operator.go

```diff
@@ -104,6 +108,9 @@ func run(cmd *cobra.Command, args []string) {
 		setupLog.Error(err, "unable to create controller", "controller", "Canary")
 		os.Exit(1)
 	}
+	pushServers, _ := cmd.Flags().GetStringSlice("push-servers")
```

Comment:
why not declare pushServers a variable director and read the flag directly into it ?

Reply:
I did that previously and declared the Servers var in the push package which I was reading in the `AddServers` func. But your last comment that it can't be aware made me confused a bit so not really sure where should the var needs to be defined